fractions/microprofile/microprofile/pom.xml

```diff
@@ -125,4 +121,21 @@
     </dependency>
   </dependencies>
 
+  <profiles>
```

Comment:
Why was this added?

The entirety of MP should be part of product

Reply:
Hm, sorry for wrong approval. Now I'm a little bit confused. What MP do we acutally implement in community/product? When looking at MP APIs versions it's like a mix of MicroProfile 1.4 (config, jwt) and 1.3 (openapi, rest-client, opentracing, FT, health, metrics).